govc/fields/set.go

```diff
@@ -0,0 +1,95 @@
+/*
+Copyright (c) 2014 VMware, Inc. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package fields
+
+import (
+	"flag"
+
+	"github.com/vmware/govmomi/govc/cli"
+	"github.com/vmware/govmomi/govc/flags"
+	"github.com/vmware/govmomi/list"
+	"github.com/vmware/govmomi/object"
+	"golang.org/x/net/context"
+)
+
+type set struct {
+	*flags.DatacenterFlag
+}
+
+func init() {
+	cli.Register("fields.set", &set{})
+}
+
+func (cmd *set) Register(f *flag.FlagSet) {}
+
+func (cmd *set) Process() error { return nil }
+
+func (cmd *set) Usage() string {
+	return "KEY VALUE PATH..."
```

Comment:
If the changes above are also done then this should also be `NAME` instead of `KEY`.